docs/Components/OptionList.md

```diff
@@ -0,0 +1,77 @@
+# OptionList
+
+The option list component lets you create a list of grouped items that merchants can pick from.
+
+## Examples
+
+#### Vanilla
+
+```js
+import {ExtensionPoint, render, OptionsList} from '@shopify/argo-admin';
+
+render(ExtensionPoint.MyExtension, (root) => {
+  const options = [
+    {label: 'Red', value: 'red', disabled: false},
+    {label: 'Green', value: 'green', disabled: false},
+    {label: 'Blue', value: 'blue', disabled: false},
+  ];
+
+  const list = root.createComponent(OptionList, {});
+  root.appendChild(list, {
+    title: 'OptionList title',
+    options,
+    allowMultiple: true,
+    onChange: (value) => console.log(`option selected: ${value}`),
+  });
+
+  const cardSection = root.createComponent(CardSection, {});
+  cardSection.appendChild('This is the best extension.');
+  card.appendChild(cardSection);
+
+  root.mount();
+});
+```
+
+#### React
+
+```jsx
+import {ExtensionPoint, OptionList} from '@shopify/argo-admin';
+import {render} from '@shopify/argo-admin/react';
```

Comment:
Should be imported from `argo-admin-react` (?)

Reply:
Fixed!